realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpAuthenticationServer.java

```diff
@@ -117,6 +127,18 @@ private static URL buildActionUrl(URL authenticationUrl, String action) {
         }
     }
 
+    private static URL buildLookupUserIdUrl(URL authenticationUrl, String action, String provider, String providerId) {
+        String authURL = authenticationUrl.toExternalForm();
+        // we need the base URL without the '/auth' part
+        String baseUrlString = authURL.substring(0, authURL.indexOf(authenticationUrl.getPath()));
```

Comment:
Wouldn't it be easier to read if you use `ACTION_LOGOUT = "auth/revoke"` as a general pattern instead?